azure-cli/ad/Add-OAuthUserImpersonationScope.ps1

```diff
@@ -0,0 +1,49 @@
+function Add-OAuthUserImpersonationScope(
+    [Parameter(Mandatory = $true)]
+    [string] $spnId
+) {
+  # import utility functions
+  . "$PSScriptRoot\..\utilities\deploy.naming.ps1"
```

Comment:
What is used from deploy.naming.ps1 ?

Reply:
Good catch, seems like a copy/paste error.